user.css

```diff
@@ -426,6 +426,14 @@ html.sidebar-hide-text .GlueDropTarget span {
     --bg-color: rgba(var(--spice-rgb-text), .2);
 }
 
+.progress-bar:not(:active) .x-progressBar-progressBarBg > div:first-child > div {
+	transition: transform 0.2s ease;

```

Comment:
Isn't 0.2s too subtle?

Reply:
i just used it because it is the same as in the `.prog-tooltip` the goal was to synchronize the two, but that didn't really work. but i'll also probably change the way the `.prog-tooltop` is placed / positioned in the dom to be a child of the progress bar drag. this would eliminate the need to change its position every few ms, so maybe be less cpu intense and also look better ass currently whith my pr if you first hover the drag it comes in from the left wich i find to be kinda annoying.